src/tests/EmailRendererTests.ts

```diff
@@ -171,6 +171,9 @@ describe("Email Renderer API test", function() {
             expect(res.status).to.equal(200);
     
             expect(JSON.parse(res.text).content.html).to.be.eq(requestBody.call_id);
+
+            // We clear the cache so that we don't have any processing still running in the background
+            runner.plugin.pluginCache.clear();
```

Comment:
Using the last test to clear a global state is dangerous (what if we run only one test ? what if we add new tests ?)
Does mocha allows a global shutdown hook ?